### PR TITLE
[DOC-1354] Specify SpatialOS unreal engine fork repo release branch

### PR DESCRIPTION
**Pull request by MarkWoulfeImprobable**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Added a note and image specifying the default branch needing to be selected when cloning the spatialos unreal engine repo from third party GUIs

#### Release note
NA

#### Tests
Improdaboc serve

#### Documentation
NA

#### Primary reviewers
@ElleEss @oblm

### CODE REVIEW

**Comment by MarkWoulfeImprobable** [comment]
FYI I didn't add a page update note on the bottom yet, can i ask if we have internal documentation on what form this should take?

File: SpatialGDK/Documentation/content/get-started/build-unreal-fork.md [hunk 1]
```diff
@@ -21,10 +21,18 @@ It only takes a few minutes to set up and includes setting up a GitHub account i
 </br>
 </br>
 <img src="{{assetRoot}}assets/github404.png" style=" float: right; margin: 10px; display: block; width: 30%; padding: 20px 20x"/>
-**Note:** You **must** follow this step to use the GDK. If you have not joined the Epic Games organization on GitHub, the [SpatialOS Unreal Engine fork link](https://github.com/improbableio/UnrealEngine) returns a GitHub 404 error and you can't download it.
+**Note:** You **must** follow this step to use the GDK. If you have not joined the Epic Games organization on GitHub, the [SpatialOS Unreal Engine fork repository](https://github.com/improbableio/UnrealEngine) returns a GitHub 404 error and you can't download it.
 
 ### Step 2: Clone the fork repository
 
+You can clone the fork repository using the command line, Github Desktop, or another third-party GUI for Git.
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/get-started/build-unreal-fork.md]: "another third-party GUI" implies GitHub Desktop is a third-party tool. 

```suggestion
You can clone the fork repository using the command line, GitHub Desktop, or any third-party GUI for Git.
```

File: SpatialGDK/Documentation/content/get-started/build-unreal-fork.md [hunk 2]
```diff
@@ -21,10 +21,18 @@ It only takes a few minutes to set up and includes setting up a GitHub account i
...
+
+**Note:** When using a third-party GUI, ensure that you’re cloning the latest stable version of the repository by selecting the branch marked **default** in the **Branch** dropdown of the [SpatialOS Unreal Engine fork repository](https://github.com/improbableio/UnrealEngine).
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/get-started/build-unreal-fork.md]: Present simple is a clearer and precise instruction. 
"you're cloning" is more "I am going along with you as your friend" which makes the text more complex.
Usually, it's better to be instructional and clear than use friendly language.

Why are you telling the user to do this? What's the problem if they don't?

```suggestion
**Note:** If you use third-party GUI, ensure that you clone the latest stable version of the repository by selecting the branch marked **default** in the **Branch** dropdown of the [SpatialOS Unreal Engine fork repository](https://github.com/improbableio/UnrealEngine). 
```

File: SpatialGDK/Documentation/content/get-started/build-unreal-fork.md [hunk 3]
```diff
@@ -21,10 +21,18 @@ It only takes a few minutes to set up and includes setting up a GitHub account i
...
+
+![img]({{assetRoot}}assets/screen-grabs/unreal-fork-repo-branch.png)
+
+_Image: The default branch you need to clone._
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/get-started/build-unreal-fork.md]: Please can we see this rendered? (Either a screen grab or on staging.)

Comment (MarkWoulfeImprobable) [reply SpatialGDK/Documentation/content/get-started/build-unreal-fork.md]: image

Will make smaller.

**Comment by ernopp** [comment]
thanks for the PR

**Pull request merged**